<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VHUpdateTips</class>
 <widget class="QDialog" name="VHUpdateTips">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>45</number>
   </property>
   <property name="topMargin">
    <number>60</number>
   </property>
   <property name="rightMargin">
    <number>50</number>
   </property>
   <property name="bottomMargin">
    <number>45</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>10</number>
   </property>
   <item row="2" column="0" colspan="5">
    <widget class="QTextEdit" name="textEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-bottom:32;
padding-left:0;
margin-left:0;
border:0;
background-color:transparent;
color:#CCCCCC;
font: 13px &quot;宋体&quot;;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:13px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:13px;&quot;&gt;1.直播插件增加问答功能；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:13px;&quot;&gt;1.直播插件增加问答功能；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:13px;&quot;&gt;1.直播插件增加问答功能；&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="btnCloseButton">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="toolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLabel" name="label_version">
     <property name="styleSheet">
      <string notr="true">QLabel{
  font: 14px &quot;宋体&quot;;
color: #dddddd;
}

</string>
     </property>
     <property name="text">
      <string>V1.7.0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>167</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/toolTip/img/toolTip/findNew.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>5</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>5</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>5</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>5</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>5</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>5</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>5</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>5</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>5</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>5</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>5</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>5</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../VhallLive/project/VhallLive.qrc"/>
 </resources>
 <connections/>
</ui>
